### ISSUE TITLE
Q-btn height should not adapt to its flex container

### ISSUE THREAD

**Issue by louisameline** [quasarframework/quasar#6069]
**Describe the bug**
Any custom `height` set on buttons in flex row containers is broken since v1.5.9, as it's overriden by this:
 ```css
.row > .q-btn height: 100%
```
and also
 ```css
.row.items-start > .q-btn, .row.items-end > .q-btn, .row.items-center > .q-btn, .row.items-baseline > .q-btn, .row.items-stretch > .q-btn, .row.q-toolbar > .q-btn, .row.q-bar > .q-btn {
    height: auto;
}
```

**Codepen/jsFiddle/Codesandbox (required)**
Just reading the CSS above makes it obvious, but if you want to see it in action:
https://codepen.io/Louis99/pen/abzJNPN

**To Reproduce**
Try to set a CSS height on any button inside a flex row

**Expected behavior**
The height of a button shouldn't adopt the height of its container by default.
Furthermore, a height set in CSS by the developer shouldn't be overriden by a bugfix which was made for an obsolete version of Chrome, see #5843

**Platform (please complete the following information):**
Any

Thank you

**Comment by rstoenescu** [comment]
Fix is already pushed.
Will be available in "quasar" v1.7.0.

**Comment by louisameline** [comment]
Great, thank you
